testsuite/tests/performance/rhoam/test_rhoam_multiple_20m.py

```diff
@@ -53,6 +53,12 @@ def number_of_apps():
     return NUMBER_OF_APPLICATIONS
 
 
+@pytest.fixture(scope='module')
+def number_of_agents():
+    """Number of Hyperfoil agents to be spawned"""
+    return 2
```

Comment:
Why does rhoam use two agents now?

Reply:
It brings more stability to tests, also RHOAM itself uses a special config for agents so it should not have an influence on their test runs.